### PR TITLE
rewrite the Control node's class description, improve some others

### PR DESCRIPTION
**Pull request by NathanLovato**
rewrite main class descriptioin
improve some signals and methods

### CODE REVIEW

File: doc/base/classes.xml [hunk 1]
```diff
@@ -13836,7 +13836,7 @@
 		</signal>
 		<signal name="minimum_size_changed">
 			<description>
-				Emitted when the minimum size of the control changed.
+				Emitted when the node's minimum size of changes.
```
Comment (Noshyaar) [inline doc/base/classes.xml]: Typo?

File: doc/base/classes.xml [hunk 2]
```diff
@@ -13820,12 +13820,12 @@
...
 		<signal name="focus_entered">
 			<description>
-				Emitted when keyboard focus is gained.
+				Emitted when the node gains keyboard focus.
 			</description>
 		</signal>
 		<signal name="focus_exited">
 			<description>
-				Emitted when the keyboard focus is lost.
+				Emitted when the node loses keyboard focus
```
Comment (Noshyaar) [inline doc/base/classes.xml]: A period is missing?

File: doc/base/classes.xml [hunk 3]
```diff
@@ -12934,23 +12934,23 @@
...
+		Godot sends Input events to the root node first, via [method Node._input]. The method distributes it through the node tree and delivers the input events to the node under the mouse cursor or on focus with the keyboard. To do so, it calls [method MainLoop._input_event]. No need to enable [method Node.set_process_input] on Controls to receive input events. Call [method accept_event] to ensure no other node receives the event, not even [method Node._unhandled_input].
+		Only the one Control node in focus receives keyboard events. To do so, the Control must get the focus mode with [method set_focus_mode]. It loses focus when another Control gets it, or if the current Control in focus is hidden.
+		You'll sometimes want Controls to ignore mouse or touch events. For example, if you place an icon on top of a button. Call [method set_ignore_mouse] for that.
+		A [Theme] resources change the Control's appearance. If you change the [Theme] on a parent Control node, it will propagate to all of its children. You can override parts of the theme on each Control with the add_*_override methods, like [method add_font_override]. You can also override the theme from the editor.
```
Comment (Noshyaar) [inline doc/base/classes.xml]: *A* [Theme] resources

**Comment by NathanLovato** [comment]
Thanks @Noshyaar, just did an amend to the PR to fix these typos.

**Pull request merged**